### ISSUE TITLE
Windows error: AttributeError: 'NoneType' object has no attribute 'TestNegativeResponse'

### ISSUE THREAD

**Issue by GoogleCodeExporter** [jeevandongre/namebench#111]
```
What steps will reproduce the problem?
1.Start Benchmark
2.
3.

What is the expected output? What do you see instead?
Error message after starting benchmark. No results.

What version of the product are you using? On what operating system?
Namebench 1.2, Windows 7 Home Premium 32 bit

Please provide any additional information below.

This is the error I get. Thinking its either a windows 7 problem or a 
mobile broadband problem...

---------------------------
'NoneType' object has no attribute 'TestNegativeResponse'
---------------------------
  File "libnamebench\nameserver_list.pyc", line 143, in __init__

  File "libnamebench\nameserver_list.pyc", line 234, in 
ApplyCongestionFactor

  File "libnamebench\conn_quality.pyc", line 102, in CheckConnectionQuality

  File "libnamebench\conn_quality.pyc", line 80, in 
GetNegativeResponseDuration

---------------------------
OK   
---------------------------


```

Original issue reported on code.google.com by `marcus.registradus@gmail.com` on 24 Mar 2010 at 12:43

**Comment by GoogleCodeExporter** [comment]
```
I experienced the exact same error, using namebench 1.2 on Windows Vista, using 
a 
Huawei HSPA modem. I can run e.g. WinMTR and other tools. I was running the 
application 
in Administrator-mode and with stopped firewall.
```

Original comment by `ripper...@gmail.com` on 7 Apr 2010 at 10:44

**Comment by GoogleCodeExporter** [comment]
```
Forgot to attach complete traceback.

Traceback (most recent call last):
  File "libnamebench\tk.pyc", line 107, in run
  File "libnamebench\base_ui.pyc", line 87, in PrepareNameServers
  File "libnamebench\nameserver_list.pyc", line 143, in __init__
  File "libnamebench\nameserver_list.pyc", line 234, in ApplyCongestionFactor
  File "libnamebench\conn_quality.pyc", line 102, in CheckConnectionQuality
  File "libnamebench\conn_quality.pyc", line 80, in GetNegativeResponseDuration
AttributeError: 'NoneType' object has no attribute 'TestNegativeResponse'
Traceback (most recent call last):
  File "libnamebench\tk.pyc", line 107, in run
  File "libnamebench\base_ui.pyc", line 87, in PrepareNameServers
  File "libnamebench\nameserver_list.pyc", line 143, in __init__
  File "libnamebench\nameserver_list.pyc", line 234, in ApplyCongestionFactor
  File "libnamebench\conn_quality.pyc", line 102, in CheckConnectionQuality
  File "libnamebench\conn_quality.pyc", line 80, in GetNegativeResponseDuration
AttributeError: 'NoneType' object has no attribute 'TestNegativeResponse'
Traceback (most recent call last):
  File "libnamebench\tk.pyc", line 107, in run
  File "libnamebench\base_ui.pyc", line 87, in PrepareNameServers
  File "libnamebench\nameserver_list.pyc", line 143, in __init__
  File "libnamebench\nameserver_list.pyc", line 234, in ApplyCongestionFactor
  File "libnamebench\conn_quality.pyc", line 102, in CheckConnectionQuality
  File "libnamebench\conn_quality.pyc", line 80, in GetNegativeResponseDuration
AttributeError: 'NoneType' object has no attribute 'TestNegativeResponse'
Traceback (most recent call last):
  File "libnamebench\tk.pyc", line 107, in run
  File "libnamebench\base_ui.pyc", line 87, in PrepareNameServers
  File "libnamebench\nameserver_list.pyc", line 143, in __init__
  File "libnamebench\nameserver_list.pyc", line 234, in ApplyCongestionFactor
  File "libnamebench\conn_quality.pyc", line 102, in CheckConnectionQuality
  File "libnamebench\conn_quality.pyc", line 80, in GetNegativeResponseDuration
AttributeError: 'NoneType' object has no attribute 'TestNegativeResponse'
Traceback (most recent call last):
  File "libnamebench\tk.pyc", line 107, in run
  File "libnamebench\base_ui.pyc", line 87, in PrepareNameServers
  File "libnamebench\nameserver_list.pyc", line 143, in __init__
  File "libnamebench\nameserver_list.pyc", line 234, in ApplyCongestionFactor
  File "libnamebench\conn_quality.pyc", line 102, in CheckConnectionQuality
  File "libnamebench\conn_quality.pyc", line 80, in GetNegativeResponseDuration
AttributeError: 'NoneType' object has no attribute 'TestNegativeResponse'
```

Original comment by `ripper...@gmail.com` on 7 Apr 2010 at 10:46

**Comment by GoogleCodeExporter** [comment]
```
i have the same problem. namebench 1.2, Windows 7 Ultimate x64
```

Original comment by `sna...@gmail.com` on 22 Apr 2010 at 5:28

**Comment by GoogleCodeExporter** [comment]
```
yeah Huawei modem for me too E1762
```

Original comment by `marcus.registradus@gmail.com` on 28 Apr 2010 at 5:48

**Comment by GoogleCodeExporter** [comment]
```
Fix is in revision 858 - For whatever reason, the DNS library was unable to 
determine your currently used 
nameservers, and was unable to detect connection quality from it.

```

Original comment by `helixblue` on 17 May 2010 at 3:30
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
Original comment by `helixblue` on 17 May 2010 at 3:30
- Changed title: **Windows error: AttributeError: 'NoneType' object has no attribute 'TestNegativeResponse'**

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
i have d same problem...
i m using win vista. broadband modem.
```

Original comment by `afaisal...@gmail.com` on 17 May 2010 at 4:40

**Comment by GoogleCodeExporter** [comment]
```
Can you try the newly released namebench 1.3-BETA1 to see if it resolves this 
bug?

```

Original comment by `helixblue` on 20 May 2010 at 3:21

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 1.3.
```

Original comment by `helixblue` on 31 May 2010 at 6:44
- Changed state: **Fixed**
